src/styles/stars.css

```diff
@@ -0,0 +1,32 @@
+.card__stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(/images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars :last-child {
+  background-image: url(/images/star.svg);
+}
+
+.card__stars-text {
```

Comment:
Stars should work correctly with modifiers
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/1856d73d-bcb7-4c62-8f1f-abcde713108e)